src/Raven.Server/Dashboard/MachineResources.cs

```diff
@@ -27,6 +27,8 @@ public class MachineResources : AbstractDashboardNotification
 
         public bool IsLowMemory { get; set; }
 
+        public bool ExtremelyLow { get; set; }
```

Comment:
Instead of `IsLowMemory` and `ExtremelyLow`, just `LowMemSeverity`

Reply:
Still need to make the change here